### PR TITLE
Fl absentee render bug

### PR DESCRIPTION
**Pull request by danielglauser**
One commit to fix the bug and one to stop hiding exceptions. Not sure why we were hiding them in the first place.

### CODE REVIEW

File: src/form_works/render.clj [hunk 1]
```diff
@@ -103,11 +104,7 @@
             .getDictionary
             (.getDictionaryObject "Fields")
             .clear)
-        (.save doc output)
-        (catch IOException e
-          (str "Error: can't add non-simple fonts, this is a constraint of PDFBox."))
-        (catch NullPointerException e
-          (str "Error: non existent field provided"))))))
+        (.save doc output)))))
```
Comment (cap10morgan) [inline src/form_works/render.clj]: If we're not going to catch anything (or use a `finally` or any other `try` features), should we pull all this code out of the `try` form?

**Comment by danielglauser** [comment]
We tested this locally with the FL absentee form. First we caused the error we saw in prod, made the changes to the code, and showed it working. Tried with the MO form too as we were successful.

File: src/form_works/render.clj (hunk 1 above)
Comment (danielglauser) [inline src/form_works/render.clj]: Agreed!

**Comment by cap10morgan** [comment]
Sounds like it fixes things and doesn't seem to break other things, so with about as much insight into how PDFs work (little to none) as anyone else, this LGTM!

**Pull request merged**